chfunds.permitted-roles.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * This constant DRUPAL_ROLE_PERMISSIONS holds the drupal role and the respective with CH Permissions in serialised format
+ * If you unserialize the contstant it will give following array in ['drupal-role-name' => [permission1, permission2 ...]] format,
+ * where 'drupal-role-name' is case insensitive but permissions aren't
+ *
+ * unserialize(DRUPAL_ROLE_PERMISSIONS) = [
+ *   'administrator' => [
+ *     'CH admin miscellaneous',
+ *     'assign CH Fund',
+ *    ],
+ *  ];
+ *
+ * which means 'adminstrator' role is given 'CH admin miscellaneous' and 'assign CH Fund' permission.
+ * You can change it to whichever role(s) you like to assign permission with. Like if you want to give
```

Comment:
Copyedit suggestion: 'You can change it to assigning whichever permissions you want to whichever roles you want, assuming permissions and roles exist. Like if you want to give'